### PR TITLE
PRO-3549 download check your answers download added to page

### PR DESCRIPTION
**Pull request by Wivenhoe-HMCTS**
### Change description ###
Add service call to PDF client in order to generate PDF file from JSON supplied by check answers summary page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/functionalTest/resources/json/checkAnswersMultipleExecutorsSummary.json [hunk 1]
```diff
@@ -0,0 +1,273 @@
+{
```
Comment (sanjayparekh-hmcts) [inline src/functionalTest/resources/json/checkAnswersMultipleExecutorsSummary.json]: Perhaps we should add Applicant Alias details here too in our test
Also, must merge from master

File: src/main/java/uk/gov/hmcts/probate/services/businessdocuments/FileSystemResourceService.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                });
+    }
+
+    public String getFileFromResourceAsString(String resourcePath) {
+        Optional<FileSystemResource> fileSystemResource = getFileSystemResource(resourcePath);
+        if (fileSystemResource.isPresent()) {
+            try {
+                return FileUtils.readFileToString(fileSystemResource.get().getFile(), Charset.defaultCharset());
+            } catch (IOException e) {
+                throw new BusinessDocumentException(BUSINESS_DOCUMENT_TEMPLATE_COULD_NOT_BE_FOUND, e);
```
[the remark this answers is not in the document]
Comment (Wivenhoe-HMCTS) [reply src/main/java/uk/gov/hmcts/probate/services/businessdocuments/FileSystemResourceService.java]: Add a new FileSystemException class

File: src/main/java/uk/gov/hmcts/probate/services/businessdocuments/PDFGenerationService.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    private Map<String, Object> asMap(String placeholderValues) {
+        try {
+
+            ObjectMapper mappy = new ObjectMapper();
+            return mappy.readValue(placeholderValues,
+                    new TypeReference<HashMap<String, Object>>() {
+                    });
+        } catch (IOException e) {
+            log.error(e.getMessage(), e);
+            throw new BusinessDocumentException(e.getMessage(), e);
```
[the remark this answers is not in the document]
Comment (Wivenhoe-HMCTS) [reply src/main/java/uk/gov/hmcts/probate/services/businessdocuments/PDFGenerationService.java]: Added a new PDFGenerationException

File: src/functionalTest/resources/json/checkAnswersMultipleExecutorsSummary.json (hunk 1 above)
Comment (Wivenhoe-HMCTS) [reply src/functionalTest/resources/json/checkAnswersMultipleExecutorsSummary.json]: added new test for alias names

**Pull request closed without merging**